test/parallel/test-dgram-send-error.js

```diff
@@ -35,3 +35,28 @@ getSocket((socket) => {
 
   socket.send('foo', socket.address().port, 'localhost', callback);
 });
+
+{
+  const socket = dgram.createSocket('udp4');
+
+  socket.on('message', common.mustNotCall('Should not receive any messages.'));
+
+  socket.bind(common.mustCall(() => {
+    const port = socket.address().port;
+    const errCode = process.binding('uv').UV_UNKNOWN;
+    const callback = common.mustCall((err) => {
+      socket.close();
+      assert.strictEqual(err.code, 'UNKNOWN');
+      assert.strictEqual(err.errno, 'UNKNOWN');
+      assert.strictEqual(err.syscall, 'send');
+      assert.strictEqual(err.address, 'localhost');
+      assert.strictEqual(err.port, port);
+    });
```

Comment:
Maybe add an assertion for the `message` format. Something like:
```js
assert.strictEqual(
    err.message,
    `${err.syscall} ${err.code} ${err.address}:${err.port} ` +
    `- Local (${err.localAddress}:${err.localPort})`
  );
```
Silly but cheap.